docs/HowTo/Monitor/Metrics.md

```diff
@@ -7,7 +7,7 @@ Use the [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled)
 alerting service to access Besu metrics. Use [Grafana](https://grafana.com/) to visualize the collected data.
 A sample [Besu Grafana dashboard](https://grafana.com/dashboards/10273) is provided. 
 
-The Besu Quickstart has [monitoring with Prometheus and Grafana configured](../../Tutorials/Quickstarts/Private-Network-Quickstart.md#monitoring-nodes-with-prometheus-and-grafana). 
+The Besu Example Networks have [monitoring with Prometheus and Grafana configured](../../Tutorials/Examples/Private-Network-Example.md#monitoring-nodes-with-prometheus-and-grafana). 
```

Comment:
Are we using `Besu Sample Networks` (see Log rotation section above) and `Besu Example Networks` interchangebly?  Suggest standardising on one (Sample Networks probably) if possible.